require/resolve.go

```diff
@@ -32,15 +34,18 @@ func (r *RequireModule) resolve(modpath string) (module *js.Object, err error) {
 	p := path.Join(start, modpath)
 	if isFileOrDirectoryPath(origPath) {
 		if module = r.modules[p]; module != nil {
+			moduleDebug(fmt.Sprintf("resolve %s (cached)", p))
```

Comment:
It's not the most efficient way of doing it. The fmt.Sprintf() should not be called if debug is disabled.